### ISSUE TITLE
Diff fails when before and after files don't match in content

### ISSUE THREAD

**Issue by Fake51** [python-openxml/opc-diag#5]
I have a repaired powerpoint file where the repair apparently consisted in removing a .xml.rel file. When trying to diff the problematic and the repaired file, opc fails with the following exception:

KeyError: "No item with name 'ppt/slideLayouts/_rels/slideLayout2.xml.rels'"

The file exists in the first file, but not in the second

**Comment by scanny** [comment]
What opc command are you running when this happens?

**Comment by Fake51** [comment]
I'm running

opc diff [problematic.pptx] [fixed.pptx]

To underline the issue, things work if I switch the order of the files - presumably because things are diffed by checking each [old-file] against [new-files]. I haven't looked into the code for it yet, apart from doublechecking that the .pptx files were actually read correctly

**Comment by scanny** [comment]
Yes, I think that's right. The code isn't sophisticated enough to deal with "package-level" differences, i.e. different members in the two zip-archives.

I vaguely remember considering a package-level diff sub-command right around the time I ran out of energy to add more to this :)

**Comment by Fake51** [comment]
Well, the tool came in handy for me today (thanks a lot, by the way), so might have a look at a patch, if I can find the time.

And thanks a lot both for responding, and doing so quickly!

**Comment by scanny** [comment]
Of course :)

`opc-diag` is a daily-use tool for me, I just got to the point where it did what I needed and I lost interest in extending it. I think the most frequent use for me is exploring an XML part in a package (.pptx for me usually) without having to unzip and reformat it and especially to clean up the file-space afterward. My favorite command is:

```bash
opc browse some.pptx somePart1.xml | vim -
```

Using this it's super fast to get into a code-folded view of the XML so I can see how the elements of interest are structured, either by PowerPoint at analysis time or me at implementation time. It's also handy at debugging time to diagnose a file that's behaving unexpectedly :)
